scripts/build/parse_blog_tags.py

```diff
@@ -28,7 +28,7 @@
                     # Otherwise, add the tags after the first line of front-matter
                     data[2] = 'tags: ["' + tag_name + '"]\n' + data[2] 
 
-                with open("src/main/content/_posts/" + file_name, 'w') as f_write:
+                with open("src/main/content/_i18n/en/_posts/" + file_name, 'w') as f_write:
                     f_write.writelines(data)
                     f_write.close()
 f.close()
```

Comment:
```suggestion
f.close()

```

Add EOL to make GitHub diff view happy

Reply:
Ah...did it break with the translation work being done on the website? I wondered what had changed. :)

Reply:
Yes, the translated posts were moved to a different location in the files, so the old blogs were being tagged while the translated blogs weren't.